setup.py

```diff
@@ -0,0 +1,65 @@
+import os.path
+from setuptools import find_packages, setup
+
+# Package data
+# ------------
+_author       = 'covjson'
+_authorEmail  = 'resc@reading.ac.uk'
+_classifiers  = [
+    'Environment :: Console',
+    'Intended Audience :: Developers',
+    'Intended Audience :: Information Technology',
+    'Intended Audience :: Science/Research',
+    'Topic :: Scientific/Engineering',
+    'Development Status :: 4 - Beta',
+    'License :: Copyright :: University of Reading',
+    'Operating System :: OS Independent',
+    'Programming Language :: Python',
+    'Topic :: Internet :: WWW/HTTP',
+    'Topic :: Software Development :: Libraries :: Python Modules',
+]
+_description  = 'Create CovJSON files from common scientific data formats'
+_downloadURL  = 'http://pypi.python.org/pypi/pycovjson/'
+_requirements = ["", "",""]
+_keywords     = ['dataset', 'coverage', 'covjson']
+_license      = 'Copyright :: University of Reading'
+_long_description    = 'A python utility library for creating CovJSON files from common scientific data formats'
+_name         = 'pycovjson'
+_namespaces   = []
+_testSuite    = 'pycovjson.test'
+_url          = 'https://github.com/Reading-eScience-Centre/pycovjson'
+_version      = '1.0.0'
+_zipSafe      = True
```

Comment:
Nice one. I will log another issue for documentation and another for CLI

On Aug 19, 2016 6:08 AM, "RileyWilliams" <notifications@github.com> wrote:

> In setup.py
> <https://github.com/Reading-eScience-Centre/pycovjson/pull/6#discussion_r75478084>
> :
>
> > +    'Programming Language :: Python',
> > +    'Topic :: Internet :: WWW/HTTP',
> > +    'Topic :: Software Development :: Libraries :: Python Modules',
> > +]
> > +_description  = 'Create CovJSON files from common scientific data formats'
> > +_downloadURL  = 'http://pypi.python.org/pypi/pycovjson/'
> > +_requirements = ["", "",""]
> > +_keywords     = ['dataset', 'coverage', 'covjson']
> > +_license      = 'Copyright :: University of Reading'
> > +_long_description    = 'A python utility library for creating CovJSON files from common scientific data formats'
> > +_name         = 'pycovjson'
> > +_namespaces   = []
> > +_testSuite    = 'pycovjson.test'
> > +_url          = 'https://github.com/Reading-eScience-Centre/pycovjson'
> > +_version      = '1.0.0'
> > +_zipSafe      = True
>
> I have rewritten the code so it no longer loads in json templates.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Reading-eScience-Centre/pycovjson/pull/6/files/70705b8f9e214f1389e1c1a9a2e8f6ff96a04e4f#r75478084>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABHJlzooirdtKoCI-_7mr2faYTOJi8Jiks5qharmgaJpZM4Jlx67>
> .
>